### ISSUE TITLE
Performance and online activity graph broken.

### ISSUE THREAD

**Issue by Duoquote** [plan-player-analytics/Plan#1425]
### Describe the issue
I just installed Plan and in performance and online activity section it says `Failed to load graph data: Unexpected end of JSON input (See ../v1/graph?type=performance&server=294c1301-407f-45a9-99b9-cb762b541159)`.

### Server information
Plan-5.1 Build 505
Paper version git-Paper-1618 (MC: 1.12.2) (Implementing API version 1.12.2-R0.1-SNAPSHOT)

### Exceptions & Other Logs
There is no error in minecraft console.

### Additional information
It looks like the [json](https://codebeautify.org/jsonviewer/cb5eb892) data is valid.  Response status is 200 but there is no data indeed.

![Imgur](https://i.imgur.com/DXSqxZe.png)

**Comment by Rsl1122** [comment]
Duplicate of #1406

Caused by ad-block in Brave browser, disable ad block on the Plan website.
My guess is Brave replaces the contents of the request when blocking it leading to this error.

**Comment by ryanbr** [comment]
Hey, just trying to debug this. Can someone paste the full url that is being blocked by Brave (and other adblockers), I can Investigate a possible fix. @Rsl1122 @Duoquote

**Comment by ryanbr** [comment]
Okay,

Filter has been removed. https://github.com/easylist/easylist/commit/af432262028f624d961e27e2a3053dc70f3732ca

**Comment by Rsl1122** [comment]
Awesome, thanks @ryabbr :)

**Comment by ryanbr** [comment]
Can we have the adblock/brave code removed also @Rsl1122 ?
